### DISCUSSION TITLE
Pandora broken after using FNIS

### DISCUSSION THREAD

**Discussion by DamonPrince** [Monitor221hz/Pandora-Behaviour-Engine-Plus#265] [Q&A] [answered]
Hey, I have a problem with Pandora and hope someone can help me get it running again. I recently installed a mod that is not compatible with Pandora, and therefore, I installed FNIS. After I was done with the mod, I uninstalled FNIS (following the uninstallation instructions on the mod page) and reinstalled Pandora. However, I then noticed that most SX animations were no longer playing. Combat mods were also affected (but outside of SX scenes, there was no A-posing or similar issues). TKDodge only allowed me to dodge when I wasn't holding a weapon. After a lot of tinkering, reinstalling Pandora and the relevant mods, and deleting and restoring the meshes folder via Vortex, I reinstalled FNIS as a test, which restored the SX mod animations. I tried for a while to get Pandora running again, but since that didn't work at all, I installed Nemesis. With Nemesis, everything is working again.

My question is: WHAT HAPPENED HERE and how do I get Pandora running again?

**Comment by Monitor221hz** [comment, marked as answer]
I don't know
